fec.config.js

```diff
@@ -0,0 +1,40 @@
+const { resolve } = require('path');
+
+module.exports = {
+  appUrl: '/insights/remediations',
+  debug: true,
+  useProxy: process.env.PROXY === 'true',
+  proxyVerbose: true,
+  interceptChromeConfig: false,
+  plugins: [],
+  hotReload: true,
```

Comment:
What I meant was more like below, like 
```suggestion
  ...(process.env.HOT ? { hotReload: process.env.HOT === 'true' } : { hotReload: true }),
```